app/views/grids/_embedButton.html.erb

```diff
@@ -0,0 +1,19 @@
+<a class="grid-embed grid-embed-<%= randomSeed %>" title="Embed this table on another site."><i class="fa fa-code" style="color:#aaa;"></i></a>
+
+<script>
+(function(){
+  $(".grid-embed-<%= randomSeed %>").click(function() {
+    prompt('Use this HTML code to embed this table on another site.', '<iframe style="border:none;" width="100%" height="900px" src="//publiclab.org/embed/grid/<% if type != "notes" %><%= type.gsub('questions', 'question').gsub('activities', 'activity') %>:<% end %><%= tagname %>"></iframe>')
```

Comment:
This looks great! however, the route for a grid of `[notes:grid:coqui]` should be:

https://publiclab.org/embed/grid/notes:grid:coqui

This will embed the thumbnail version, not the text table version (which would be https://publiclab.org/embed/grid/coqui).

Does that make sense?